documentation/src/main/asciidoc/topics/code_examples/HotRodClientInstance.java

```diff
@@ -0,0 +1,12 @@
+org.infinispan.client.hotrod.configuration.ConfigurationBuilder cb
+      = new org.infinispan.client.hotrod.configuration.ConfigurationBuilder();
+cb.tcpNoDelay(true)
+  .statistics()
+      .enable()
+      .jmxEnable()
+      .jmxDomain(org.infinispan)
```

Comment:
org.infinispan is a String, so should be "org.infinispan"

Reply:
face palm. thanks for catching this.